src/styles/_card.scss

```diff
@@ -0,0 +1,93 @@
+.cards-container {
+  margin: 0 auto;
+  padding: 50px 0;
+  display: flex;
+  flex-wrap: wrap;
+  max-width: 944px;
+}
```

Comment:
there should always be 4 cards in a row (not 2, 3 or 5)
![2023-05-17_15h18_36](https://github.com/mate-academy/layout_catalog/assets/61904995/9c69fffc-9a26-4e4d-aa99-afe2a205ad34)
![2023-05-17_15h18_22](https://github.com/mate-academy/layout_catalog/assets/61904995/e5c5a554-16de-4435-869b-df2dd70c477b)